### ISSUE TITLE
X and Y values for axis

### ISSUE THREAD

**Issue by MorneJooste** [PhilJay/MPAndroidChart#2080]
Good day,

How do I give in an array of both X and Y values? I cannot find how to do this with the latest version? 

All the methods used in online tutorials does not seem to be working with com.github.PhilJay:MPAndroidChart:v3.0.0-beta1 version?

For example this tutorial http://www.truiton.com/2015/04/android-chart-example-mp-android-chart-library/

Please can someone assist me.

Thank you

**Comment by alejandrocfc** [comment]
Hi Morne, I have the same issue with version v3.0.0-beta1: My CustomXAxis class implement AxisValueFormatter but for me the value float is set twice, sometimes, the same value:

@Override
    public String getFormattedValue(float value, AxisBase axis) {
        return xAxes.get((int) value);
    }

xAxes is an ArrayString whos contain the data I wanna set in my xLabel. BTW I found not a clean solution, I set the version v2.0.9 and works like a charm. I hope this works or if u found another way let me know !! (:

PD: My english is no very good looking (:

**Comment by MorneJooste** [comment]
I did the same. I used the latest stable build and I was able to get it working to my requirements.

Thank you for your response

On Tue, Aug 02, 2016 at 6:37 AM, Alejandro Gómez < notifications@github.com [notifications@github.com] > wrote:
Hi Morne, I have the same issue with version v3.0.0-beta1: My CustomXAxis class implement AxisValueFormatter but for me the value float is set twice, sometimes, the same value:

@Override [https://github.com/Override]
public String getFormattedValue(float value, AxisBase axis) {
return xAxes.get((int) value);
}

xAxes is an ArrayString whos contain the data I wanna set in my xLabel. BTW I found not a clean solution, I set the version v2.0.9 and works like a charm. I hope this works or if u found another way let me know !! (:

PD: My english is no very good looking (:
